docs/source/app-dev/services.rst

```diff
@@ -105,6 +105,10 @@ For details on how to use command deduplication, see the :ref:`Application Archi
 
   - Command deduplication is *guaranteed* to work even if commands are submitted to different participants.
 
+  The following ledger integrations support command deduplication across participants:
+    - `Daml Driver for VMware Blockchain <https://www.digitalasset.com/daml-for-vmware-blockchain/>`__
+    - :ref:`Daml Sandbox <sandbox-manual>`
```

Comment:
There really is only one single participant. So while the statement technically is true, it's not very helpful.

Reply:
You mean for Daml Sandbox, right? I've removed that and put this note above where we mention 'command deduplication across participants'.